interface-definitions/service-mdns-repeater.xml.in

```diff
@@ -15,6 +15,31 @@
             <children>
               #include <include/generic-disable-node.xml.i>
               #include <include/generic-interface-multi.xml.i>
+              <leafNode name="ip-version">
+                <properties>
+                  <help>IP address version to use</help>
+                  <valueHelp>
+                    <format>_ipv4</format>
```

Comment:
Stray underscore?

Reply:
This is intentional so that CLI completion shows literal `ipv4` instead of expanding to pattern `<x.x.x.x>`.

With underscore, the shell shows this (what we want):
```
Possible completions:
   ipv4                 Use only IPv4 address
   ipv6                 Use only IPv6 address
   both                 Use both IPv4 and IPv6 address
```
instead of this (what we do not want):
```
Possible completions:
   <x.x.x.x>            Use only IPv4 address
   <h:h:h:h:h:h:h:h>    Use only IPv6 address
   both                 Use both IPv4 and IPv6 address
```
The relevant code is [here](https://github.com/vyos/vyatta-cfg/blob/f86d305d2cd638e70e64ef912d826aa1c448c29b/etc/bash_completion.d/vyatta-cfg#L478-L480).

I'd be open to suggestions if there is there any other way to achieve this.

Reply:
@c-po, this discovery was based on your hint on Slack a few months back :)
Otherwise, I'd have no clue about where to even look for in the code base.